mpeg2dec.c

```diff
@@ -1685,25 +1685,19 @@ int stream_component_open(VideoState *is, int stream_index)
 #endif
         }
 
-        if (codecCtx->codec_id == AV_CODEC_ID_H264) {

```

Comment:
No need to change this

Reply:
It's tied to the line below - but I had to add the use_cuvid check ... or cuvid is blocked if H.264 => so OK?